### PR TITLE
update error code

### PR DESCRIPTION
**Pull request by cooper-lzy**
https://github.com/vesoft-inc/nebula-ent-docs/issues/415

### CODE REVIEW

**Comment by cooper-lzy** [comment]
@dutor The following error codes need to be interpreted：
E_HISTORY_CONFLICT: 
E_PRIVILEGE_ALL_TAG_EDGE_SETTLED:
E_NO_TRANSFORMED:

File: docs-2.0/20.appendix/error-code.md [hunk 1]
```diff
@@ -54,44 +50,51 @@
...
 |`E_WRONGCLUSTER`|`-2010`| 错误的集群|
-|`E_ZONE_NOT_ENOUGH`|`-2011`| listener 冲突|
-|`E_ZONE_IS_EMPTY`|`-2012`| 主机不存在|
-|`E_SCHEMA_NAME_EXISTS`|`-2013`| Schema 名字已存在|
-|`E_RELATED_INDEX_EXISTS`|`-2014`| 与 Tag 或 Edge Type 相关的索引存在，不能被删除 |
-|`E_RELATED_SPACE_EXISTS`|`-2015`| 仍有图空间在主机上，不能被删除|
+|`E_LISTENER_CONFLICT`|`-2011`| listener 冲突|
+|`E_ZONE_NOT_ENOUGH`|`-2012`| 主机数量不足|
+|`E_ZONE_IS_EMPTY`|`-2013`| Zone 中没有主机|
+|`E_HISTORY_CONFLICT`|`-2018`| 和已存在的对象冲突 |
```
Comment (dutor) [inline docs-2.0/20.appendix/error-code.md]: 与历史命名冲突

File: docs-2.0/20.appendix/error-code.md [hunk 2]
```diff
@@ -123,6 +124,7 @@
 |`E_TRANSFER_LEADER_FAILED`|`-3036`| leader 转换失败|
 |`E_INVALID_STAT_TYPE`|`-3037`| 无效的统计类型|
 |`E_INVALID_VID`|`-3038`| VID 无效|
+|`E_NO_TRANSFORMED`|`-3039`| 未转换的 |
```
Comment (dutor) [inline docs-2.0/20.appendix/error-code.md]: This code has not been used anywhere, you could just ignore it.

File: docs-2.0/20.appendix/error-code.md [hunk 3]
```diff
@@ -54,44 +50,51 @@
...
-|`E_CORRUPTED_BALANCE_PLAN`|`-2027`| 已经损坏的数据均衡计划|
+|`E_CORRUPTED_BALANCE_PLAN`|`-2027`| 已经损坏的负载均衡计划|
+|`E_NO_INVALID_BALANCE_PLAN`|`-2028`| 没有无效的负载均衡计划 |
 |`E_IMPROPER_ROLE`|`-2030`| 回收用户角色失败|
 |`E_INVALID_PARTITION_NUM`|`-2031`| 无效的分区数量|
 |`E_INVALID_REPLICA_FACTOR`|`-2032`| 无效的副本因子|
 |`E_INVALID_CHARSET`|`-2033`| 无效的字符集|
 |`E_INVALID_COLLATE`|`-2034`| 无效的字符排序规则|
 |`E_CHARSET_COLLATE_NOT_MATCH`|`-2035`| 字符集和字符排序规则不匹配|
+|`E_PRIVILEGE_ALL_TAG_EDGE_SETTLED`|`-2036`| 授权或撤销权限前需要删除所有的 Tag 或 Edge type |
```
Comment (dutor) [inline docs-2.0/20.appendix/error-code.md:76]: LGTM.

**Pull request merged**